### PR TITLE
Create HoneyBot.yml

### PR DESCRIPTION
**Pull request by Abdur-rahmaanJ**

### CODE REVIEW

File: _data/projects/HoneyBot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: HoneyBot
+desc: python powered OOP modular (plugins-supported) IRC bot. ignited in Mauritius.
+site:
```
Comment (ritwik12) [inline _data/projects/HoneyBot.yml]: @Abdur-rahmaanJ Please add link to your repo here

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/HoneyBot.yml (hunk 1 above)
Comment (Abdur-rahmaanJ) [reply _data/projects/HoneyBot.yml]: ok

**Comment by Abdur-rahmaanJ** [comment]
@ritwik12 done

File: _data/projects/HoneyBot.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: HoneyBot
+desc: python powered OOP modular (plugins-supported) IRC bot. ignited in Mauritius.
+site: https://github.com/Abdur-rahmaanJ/honeybot
+tags:
+- python
+- irc
+- bot
+upforgrabs:
+  name: Help wanted
```
Comment (ritwik12) [inline _data/projects/HoneyBot.yml]: @Abdur-rahmaanJ Your label name is "help wanted" as I can see. So, please change this to lower case.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by Abdur-rahmaanJ** [comment]
letter h changed

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@Abdur-rahmaanJ Sorry for the delay, Thanks!!

**Pull request merged**